resources/css/treeflex.css

```
@@ -0,0 +1,22 @@
+@import 'treeflex/dist/css/treeflex.css';
+
+.tf-tree .tf-nc {
+    @apply border-none p-0 flex flex-col;
+    width: 300px;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)